build_tools/install_rocm_from_artifacts.py

```diff
@@ -206,6 +206,8 @@ def retrieve_artifacts_by_run_id(args):
             extra_artifacts.append("rand")
         if args.rccl:
             extra_artifacts.append("rccl")
+        if args.rocprofiler_compute:
+            extra_artifacts.append("rocprofiler-compute")
```

Comment:
Might we want to have a single `args.rocprofiler` that fetches rocprofiler-compute, rocprofiler-systems, rocprofiler-sdk, and any other similar artifacts?

Fine to map 1:1 explicitly as you have it here, I'm just curious.